README.md

```diff
@@ -46,6 +46,15 @@ To build the plugin from code:
 At this point the you can run TeamCity from IntelliJ. If need be you can run  TeamCity in debug mode and step 
 through the plugin code as it is executed by TeamCity.
 
+## Updating the version of Octopus CLI we embed
```

Comment:
Given what we confirmed about pre-installing the platform specific version in the path, the tools thing has less appeal, given the effort it would take and the migration it would cause.
Automating this in the future might make a lot of sense. Although you have to do the same steps to test a CLI version in a local TeamCity instance anyway, you might as well just do the commit at the end and know it's got the right version.